crossdock/client/timeout/behavior.go

```diff
@@ -48,15 +55,49 @@ func Run(t crossdock.T) {
 
 	ch := raw.New(dispatcher.Channel("yarpc-test"))
 	_, _, err := ch.Call(yarpc.NewReqMeta(ctx).Procedure("sleep/raw"), nil)
-	fatals.Error(err, "expected a failure for timeout")
+	fatals.Error(err, "expected an error")
 
 	if transport.IsBadRequestError(err) {
 		t.Skipf("sleep/raw method not implemented: %v", err)
 		return
 	}
 
+	assert.True(transport.IsTimeoutError(err),
+		"expected a TimeoutError, got %T", err)
+
 	form := strings.HasPrefix(err.Error(), `timeout for procedure "sleep/raw" of service "yarpc-test" after`)
 	assert.True(form, "error message has expected prefix for timeouts, got %q", err.Error())
-	assert.True(
-		transport.IsTimeoutError(err), "error should be a TimeoutError, got %T", err)
+
+	form = strings.HasSuffix(err.Error(), `(timeout)`)
+	assert.True(form, "error message has expected suffix for timeouts, got %q", err.Error())
+}
+
+func remoteTimeout(t crossdock.T) {
+	assert := crossdock.Assert(t)
+	fatals := crossdock.Fatals(t)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 200*time.Millisecond)
+	defer cancel()
+
+	dispatcher := disp.Create(t)
+	fatals.NoError(dispatcher.Start(), "could not start Dispatcher")
+	defer dispatcher.Stop()
+
+	ch := raw.New(dispatcher.Channel("yarpc-test"))
+	_, _, err := ch.Call(yarpc.NewReqMeta(ctx).Procedure("timeoutshort/raw"), nil)
```

Comment:
For later: We should probably combine these sleep methods on the subject to take the sleep duration as a parameter.